### ISSUE TITLE
[onert] Suspicious logic in cker optimized quantized brodcast binary arithmetic kernels

### ISSUE THREAD

**Issue by krayzemli** [Samsung/ONE#4540]
Browsing the code I noticed that BinaryBroadcastFiveFold swaps arguments, so that broadcast tensor is always the first one. This is ok for commutative float operations, but quantized ops may use different quantization settings for inputs, and these settings are not swapped together with tensor data. This needs checking. If I'm right, this may also mean that the test set for binary arithmetics is incomplete.

**Comment by glistening** [comment]
@krayzemli `BinaryBroadcastFiveFold` seems to be used `Add` and `Mul` only.

**Comment by krayzemli** [comment]
Both BroadcastAddDispatchQuant8 and BroadcastMulDispatchQuant8 use BroadcastBinaryArithmeticOp with inputs (but not quantization) swapping, and both are used in BroadcastBinaryArithmeticOp specialization for uint8_t. BTW, quantized SUB is  implemented by using ADD and negating quantization multiplicator of the second argument, and so, quantized broadcast SUB would also break when the second argument is the one to be broadcasted, even if quantization of both arguments is the same,

**Comment by periannath** [comment]
@krayzemli 

> but quantized ops may use different quantization settings for inputs, and these settings are not swapped together with tensor data.

You're right. Quantization params should be swapped along with tensor data. There is a nnapi test which tests subtraction of two tensors with different scales (`sub_quantized_different_scales.mod.py`). I think we should enable this test.
